src/server/http/__tests__/xsrf.js

```diff
@@ -71,6 +85,24 @@ describe('xsrf request filter', function () {
         if (method === 'HEAD') expect(resp.payload).to.be.empty();
         else expect(resp.payload).to.be('ok');
       });
+
+      it('accepts requests with any content-type header', async function () {
+        const resp = await inject(kbnServer, {
+          url: '/xsrf/test/route',
+          method: method,
+          headers: {
+            [contentTypeHeader]: 'anything',
+          },
+        });
+
+        expect(resp.statusCode).to.be(200);
+
+        if (method === 'HEAD') {
```

Comment:
Yeah, me too, it's just a copy-paste of the existing test. Do you want me to change the original test case as well?